### DISCUSSION TITLE
Can't change sensitivity on motion detection

### DISCUSSION THREAD

**Discussion by IjvenR** [OSNVR/OS-NVR#2] [General]
Hi,
I like the software so far, I just have one little issue. 
When the sensitivity is changed in the setting and saved, when going back in the motion detection settings the default value (8) is reset. 

This is quite the issue as this only permits activity recording for approximately 1/3 of the room (somebody walking in halfway and turning around is not detected).

Is there any setting that I'm overlooking, or is this a bug? 

Looking forward to your response!

Kind regards, 
Ijven

**Comment by Curid** [comment]
This should be fixed in [v0.13.2](https://codeberg.org/Curid/os-nvr/releases/tag/v0.13.2)

Sorry for the late replay, Github didn't notify me.
